### DISCUSSION TITLE
Is 16-bit color depth supported?

### DISCUSSION THREAD

**Discussion by ronny-nilsson** [any1/neatvnc#71] [Q&A]
Hi
Is using a 16-bit color depth supported? Changing the default fourcc DRM_FORMAT_ABGR8888 to anything "less" seems to not work.

**Comment by any1** [comment]
Buffers with 16-bit colour depth should be supported, but they are largely untested as this library has been developed against a platform that uses 32-bit buffers exclusively.

Have you tried different encoding methods? It's probably best to start with the "raw" encoding and go from there.
